UPDATING.md

```diff
@@ -24,6 +24,7 @@ assists people when migrating to a new version.
 
 ## Next
 
+- [26034](https://github.com/apache/superset/issues/26034): Fixed a problem where numeric x-axis were being treated as categorical values. As a consequence of that, the way labels are displayed might change given that ECharts has a different treatment for numerical and categorical values. To revert to the old behavior, users need to manually convert numerical columns to text so that they are treated as categories. Check https://github.com/apache/superset/issues/26159 for more details.
```

Comment:
```suggestion
- [26034](https://github.com/apache/superset/issues/26034): Fixes a problem where numeric x-axis were being treated as categorical values. As a consequence of that, the way labels are displayed might change given that ECharts has a different treatment for numerical and categorical values. To revert to the old behavior, users need to manually convert numerical columns to text so that they are treated as categories. Check https://github.com/apache/superset/issues/26159 for more details.
```

It seems like (per previous items) we define intent as opposed to what it did.